source

```diff
@@ -103133,23 +103133,28 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
    <dd>Switch to the <span>before DOCTYPE public identifier state</span>.</dd>
 
    <dt>U+0022 QUOTATION MARK (&quot;)</dt>
-   <dd><span>Parse error</span>. Set the DOCTYPE token's public identifier to the empty string (not
+   <dd>This is a <dfn data-x="parse-error-missing-whitespace-after-doctype-public-keyword">missing-whitespace-after-doctype-public-keyword</dfn>
+   <span>parse error</span>. Set the DOCTYPE token's public identifier to the empty string (not
    missing), then switch to the <span>DOCTYPE public identifier (double-quoted) state</span>.</dd>
 
    <dt>U+0027 APOSTROPHE (')</dt>
-   <dd><span>Parse error</span>. Set the DOCTYPE token's public identifier to the empty string (not
+   <dd>This is a <span data-x="parse-error-missing-whitespace-after-doctype-public-keyword">missing-whitespace-after-doctype-public-keyword</span>
+   <span>parse error</span>. Set the DOCTYPE token's public identifier to the empty string (not
    missing), then switch to the <span>DOCTYPE public identifier (single-quoted) state</span>.</dd>
 
    <dt>U+003E GREATER-THAN SIGN (&gt;)</dt>
-   <dd><span>Parse error</span>. Set the DOCTYPE token's <i data-x="force-quirks flag">force-quirks flag</i> to <i>on</i>.
+   <dd>This is an <dfn data-x="parser-error-abrupt-closing-of-doctype">abrupt-closing-of-doctype</dfn>
```

Comment:
It should be pars**e**-error-abrupt-closing-of-doctype. You also will need to replace `dfn` with `span` in `Before DOCTYPE public identifier state` for this error.